### PR TITLE
Apps up a* and b*

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/bb_accounts/models.py [hunk 1]
```diff
@@ -348,41 +344,10 @@ def funding(self):
...
-            owner=self,
-            status__slug__in=['campaign', 'done-complete', 'done-incomplete', 'voting', 'voting-done']
-        ).count()
-
-    @property
-    def has_projects(self):
-        """ Return the number of projects a user started / is owner of """
-        return get_project_model().objects.filter(owner=self).count() > 0
-
-    @property
```
Comment (aethemba) [inline bluebottle/bb_accounts/models.py]: @gannetson , @rollick , don't we need these helpers anymore?

Comment (gannetson) [inline bluebottle/bb_accounts/models.py]: Moved to bluebottle/members/models.py

File: bluebottle/bb_payouts/models.py [hunk 2]
```diff
@@ -520,30 +516,6 @@ class Meta:
         ordering = ['start_date']
         abstract = True
 
-    def _get_organization_fee(self):
-        """
```
Comment (aethemba) [inline bluebottle/bb_payouts/models.py]: Is this not used anymore?

File: bluebottle/donations/migrations/0001_initial.py [hunk 3]
```diff
@@ -16,26 +12,23 @@ class Migration(SchemaMigration):
...
-        db.create_table(MODEL_MAP['donation']['table'], (
+        db.create_table('donations_donation', (
             (u'id',
              self.gf('django.db.models.fields.AutoField')(primary_key=True)),
             ('amount',
              self.gf('django.db.models.fields.DecimalField')(max_digits=16,
                                                              decimal_places=2)),
             ('project', self.gf('django.db.models.fields.related.ForeignKey')(
-                to=orm[MODEL_MAP['project']['model']])),
+                to=orm['projects.Proejct'])),
```
Comment (aethemba) [inline bluebottle/donations/migrations/0001_initial.py]: @gannetson , typo

File: bluebottle/donations/migrations/0001_initial.py [hunk 4]
```diff
@@ -145,21 +138,21 @@ def backwards(self, orm):
             'created': ('django.db.models.fields.DateTimeField', [],
                         {'default': 'datetime.datetime.now', 'blank': 'True'}),
             'fundraiser': ('django.db.models.fields.related.ForeignKey', [], {
-            'to': "orm['{0}']".format(MODEL_MAP['fundraiser']['model']),
+            'to': "orm['fundtraisers.Fundraiser']",
```
Comment (aethemba) [inline bluebottle/donations/migrations/0001_initial.py]: @gannetson , typo

File: bluebottle/fundraisers/migrations/0003_migrate_amounts.py [hunk 5]
```diff
@@ -236,9 +233,9 @@ def backwards(self, orm):
             'why': ('django.db.models.fields.TextField', [],
                     {'max_length': '265', 'blank': 'True'})
         },
-        MODEL_MAP['organization']['model_lower']: {
+        u'organzation.orgranization': {
```
Comment (aethemba) [inline bluebottle/fundraisers/migrations/0003_migrate_amounts.py]: @gannetson , typo

File: bluebottle/fundraisers/migrations/0003_migrate_amounts.py [hunk 6]
```diff
@@ -236,9 +233,9 @@ def backwards(self, orm):
...
             'Meta': {'ordering': "['name']",
-                     'object_name': MODEL_MAP['organization']['class']},
+                     'object_name': 'Organzation'},
```
Comment (aethemba) [inline bluebottle/fundraisers/migrations/0003_migrate_amounts.py]: @gannetson , typo

File: bluebottle/payments/migrations/0001_initial.py [hunk 7]
```diff
@@ -169,8 +165,8 @@ def backwards(self, orm):
             'name': (
                 'django.db.models.fields.CharField', [], {'max_length': '100'})
         },
-        MODEL_MAP['order']['model_lower']: {
-            'Meta': {'object_name': MODEL_MAP['order']['class']},
+        u'orders.order': {
+            'Meta': {'object_name': 'ORder'},
```
Comment (aethemba) [inline bluebottle/payments/migrations/0001_initial.py]: @gannetson , typo, not sure if its an issue in this case though

File: bluebottle/bb_payouts/models.py (hunk 2 above)
Comment (gannetson) [inline bluebottle/bb_payouts/models.py]: Moved to organizations/models.py

**Pull request merged**